### ISSUE TITLE
Map the Home Assistant config directory to the add-on container

### ISSUE THREAD

**Issue by falzm** [fuslwusl/homeassistant-addon-victoriametrics#29]
Hi 👋 
Would you consider adding the Home Assistant global config directory to the mapping of your add-on container? In this PR I've introduced the ability to pass extra command line arguments such as `-promscrape.config=FILE`, which allows using alternative Prometheus server configuration – however at the moment it isn't possible to pass a file via the global `/config` Home Assistant directory. Even a read-only mapping would suffice in this case.

Basically, you'd have to add `- config` to the `map:` attribute of your config.yaml add-on file: https://github.com/fuslwusl/homeassistant-addon-victoriametrics/blob/main/victoria-metrics/config.yaml#L19

**Comment by fuslwusl** [comment]
Thanx for your PR. I think adding the config folder could lead to a security issue. At the moment victoriametrics has only access to the share folder because there should not contain any secrets. But the config folder contains sensible stuff and i wanted to avoid this. 
Maybe you could put the promscrape-config to the share folder as some sort of workaround?!

**Comment by andyfr** [comment]
I would be interested in changing the prometheus.tpl as well, share folder sounds totally fine for me. If I can help with something let me know. 
@falzm where can I find your PR? There is currently none open and I can't find something related in the closed PRs.

**Comment by falzm** [comment]
@andyfr here: #24

**Comment by fuslwusl** [comment]
@falzm Thanx for your help!
